partials/re_verify.html

```diff
@@ -0,0 +1,51 @@
+<% extends "partials/base/base.html" %>
+
+<% block content %>
+<!--Header Area-->
+<tr style="page-break-before: always">
+  <td valign="top">
+    <p class="primary" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0 0 24px 0; text-align: center;"><% block verify_content %>{{{t "You signed in to Firefox from [OS], [Location, Location, Location] Please confirm this sign-in to begin syncing with [OS]." }}}<% endblock %></p>
```

Comment:
@shane-tomlinson, @rfk This might be something that I missed, but not sure how we wanted to get location of user. Did we want show IP address and eventually do a location lookup?